resources/daemonset.yaml

```diff
@@ -7,33 +7,26 @@ spec:
   - Ephemeral
   podInfoOnMount: true
   attachRequired: false
-
 ---
-
 apiVersion: v1
 kind: Secret
 metadata:
   name: cosi.storage.k8s.io
 data:
   key: none
-
 ---
-
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: cosi.storage.k8s.io
```

Comment:
Please make naming consistent with other components. 
//cc Tejas Parikh

Reply:
We have been using `objectstorage-` as prefix for COSI resource names. Would recommend changing name to `objectstorage-adapter`.

Reply:
should I rename the csi-adapater to objectstorage-adapter everywhere?